LICENSE-3rdparty.csv

```diff
@@ -3,3 +3,4 @@ elfutils,"https://sourceware.org/elfutils/",LGPL-3.0-or-later,Free Software Foun
 libcap,"https://git.kernel.org/pub/scm/libs/libcap/libcap.git/",BSD-3-Clause,Free Software Foundation 
 liblzma,"https://github.com/kobolabs/liblzma",Public-Domain,NA
 llvm-demangle,"https://github.com/llvm/llvm-project",Apache-2.0,llvm-project
+tcbrindle/span,"https://github.com/tcbrindle/span",Boost Software License 1.0
```

Comment:
Ah yes, gcc 9 is too old so we pull in tcbrindle. 👍

Reply:
The other solution would be to update to gcc 11 on ubuntu 16.04.